pkg/volume/volume.go

```diff
@@ -452,3 +452,8 @@ type Mounter interface {
 	Mount(vol *Info, source, target, fsType string, options []string) error
 	Unmount(target string) error
 }
+
+type Expand interface {
+	NodeExpand(source, target string) error
+	ControllerExpand(ctx context.Context, vol *Info) error
+}
```

Comment:
Typically Go interfaces are named with "er" at the end. See the "Mount**er**" as an example. Please follow that and rename to `Expander`.